tutor/src/components/date-time-input.js

```diff
@@ -64,6 +64,12 @@ const DateTimeInput = (props) => useObserver(() => {
   const id = props.id || uniqueId(props.name);
   const LabelWrapper = props.labelWrapper || React.Fragment;
 
+  const onUpdateDate = dt => {
+    const ev = { target: { name: field.name, value: dt } };
+    field.onChange(ev);
+    props.onChange && props.onChange(ev);
+  };
+  
```

Comment:
FYI, I typically prefer to use `@action.bound` decorator instead of using `=>` @action.bound wraps the entire function's changes in a mobx transaction action so they're all applied at once.

Reply:
good to know @nathanstitt !